### ISSUE TITLE
qmail/maildrop config documentation - pass new mail to a script

### ISSUE THREAD

**Issue by benedetto** [Uberspace/lab#890]
**TL;DR:**
Is there a possibility in the qmail or maildrop config to hand incoming mail to a certain script or tool?

**Long version**
Since more and more online shops send invoices/receipts via e-mail I am thinking about setting up a (semi-)automated process to archive those invoices. My idea is to use [ripMIME](https://github.com/inflex/ripMIME) to extract attached pdf files from incoming mails and save them to an archive folder. I have a plan how to process the files once they are extracted from the mail – however, I am struggling to find a way to pass the mail to ripMIME to do the extraction in the first place. Unfortunately I could not find any hints or documentation on how to call and hand over an incoming mail to a tool or script.

Any help or starting point where to look for would be highly appreciated.

**Comment by thomas-kammerer** [comment]
I'm trying to set up something similar, but didn't work out yet.
`.qmail-test` could look like:
`| /home/USERNAME/ripMIME/ripmime -d /home/USERNAME/test`
`| /home/USERNAME/ripMIME/ripmime -d /home/USERNAME/test -i`

**Comment by luto** [comment]
https://github.com/Uberspace/manual/issues/12
